packages/wasm/crate/src/storage.rs

```diff
@@ -168,10 +171,7 @@ impl IndexedDBStorage {
         let store = tx.object_store(&self.constants.tables.assets)?;
 
         Ok(store
-            .get_owned(base64::Engine::encode(
-                &base64::engine::general_purpose::STANDARD,
-                id.to_proto().inner,
-            ))?
+            .get_owned(byte_array_to_base64(&id.to_proto().inner))?
```

Comment:
A lot of this file's line changes are just refactoring to use this `byte_array_to_base64` helper. The meat of the changes is down below.